src/layouts/shortcodes/subctl-install.html

```diff
@@ -0,0 +1,15 @@
+Download the subctl binary and make it available on your PATH.
```

Comment:
We can reuse this from other parts so we only need to maintain one.... the bad side is that I couldn't use the 
` ` ` bash 
` ` `

syntax.. (it didn't work), I also tried using .md extension..